packages/arb-token-bridge-ui/tests/e2e/cypress.d.ts

```diff
@@ -60,6 +63,9 @@ declare global {
       findGasFeeSummary: typeof findGasFeeSummary
       findMoveFundsButton: typeof findMoveFundsButton
       findSelectTokenButton: typeof findSelectTokenButton
+      openTransactionDetails: typeof openTransactionDetails
+      closeTransactionDetails: typeof closeTransactionDetails
+      findTransactionDetailsCustomDestinationAddress: typeof findTransactionDetailsCustomDestinationAddress
```

Comment:
NIT: Shouldn't this be `findCustomDestinationAddressInTransactionDetails` so that it's consistent with a similarly named method `findTransactionInTransactionHistory` ?